Formula/git-remote-hg.rb

```diff
@@ -0,0 +1,17 @@
+class GitRemoteHg < Formula
+  desc "Transparent bidirectional bridge between Git and Mercurial"
+  homepage "https://github.com/felipec/git-remote-hg"
+  url "https://github.com/felipec/git-remote-hg/archive/v0.3.tar.gz"
+  sha256 "2dc889b641d72f5a73c4c7d5df3b8ea788e75a7ce80f5884a7a8d2e099287dce"
+
+  depends_on :hg
+  depends_on :python
```

Comment:
Ah, that old chestnut. We `inreplace` out the `python2` for `python` in that case. We do so already for `src`, `duplicity`, probably a fair few others.

@tdsmith Just to check, is there any risk mixing Python interpreters here, if we switch our `mercurial` to use our Python & this uses the system Python instead?